GOVERNANCE.md

```diff
@@ -130,24 +130,76 @@ See "WG Membership" above.
 
 When considering adding a new releaser an email should be sent to the
 [Technical Steering Committee](https://github.com/nodejs/tsc) for approval.
-After approval the nominee will be assigned a mentor from the release team
+After approval the nominee will be assigned a mentor from the releasers team
 to help walk them through the process to learn how to prepare a release.
-The nominee will then be expected to prepare 1 release on any branch, which
-can be promoted by any other member of the release team. After this
-release the nominee will be considered a full member of the releasers
-team. The first release promoted by a new release team member must have
-a mentor from the current release team available to support during the process.
-
-At any point during this process any member of the Release WG
-can raise an objection to the TSC.
-
-When being officially added to the Releasers group the following must happen:
-
-* Added to Releasers team in the Node.js org (grants ci-release access)
-* Added to security-release team in the Node.js and nodejs-private orgs
-* SSH keys need to be added to the dist user on the www machine
-* GPG keys need to be added to the nodejs/node [README.md](https://github.com/nodejs/node/#release-team)
-* Raise an issue in [nodejs/docker-node](https://github.com/nodejs/docker-node/) to note that the gpg keys will need to be updated.
-
-New Releasers need to also be on the LTS team to do LTS releases.
-New Releasers should wait at least 2 weeks after adding credentials before signing a release.
+The nominee will then be expected to prepare one release on any active release
+line, which can be tagged, signed and promoted by any other existing member of
+the releasers team. After this release the nominee will be considered a full
+member of the releasers team. The first release promoted by a new releasers
+team member must have a mentor from the current releasers team available to
+support during the process.
+
+At any point during this process any member of the Release WG can raise an
+objection to the TSC.
+
+After being being approved by the TSC, the new releaser must:
+
+* Be added to the GitHub
+  [releasers team](https://github.com/orgs/nodejs/teams/releasers) in the
+  Node.js org (grants ci-release access)
+* Be added to the GitHub
+  [security-release team](https://github.com/orgs/nodejs/teams/security-release)
+  in the Node.js and nodejs-private orgs
+* Have a single, high quality SSH key added to the "dist" user on the primary
+  www server (see below for guidelines regarding SSH key quality)
+* Have a GPG key added to the nodejs/node
+  [README.md](https://github.com/nodejs/node/#release-team)
+* Raise an issue in
```

Comment:
This could alternatively be to send a PR to https://github.com/nodejs/docker-node/blob/master/keys/node.keys